helpwave/web
# feat: replace green confirm button with red

This pull request replaces the green confirm button with red color

Closes https://github.com/helpwave/web/issues/245

**Testing instructions:**
- Go to the organizations' page and click the edit icon of any organization from the list to see organization info.
- Under the Danger Zone section click on the Delete Organization button you will see a confirmation dialogue and it should contain a confirmation button in red color instead of green.

**Additional notes:**
- Refactored the confirmation dialogue to support different types of confirmations. The default confirmation dialogue is Positive Action.
